pkg/ui/react-app/src/pages/graph/Panel.tsx

```diff
@@ -429,23 +514,42 @@ class Panel extends Component<PanelProps & PathPrefixProps, PanelState> {
                 <option value="thanos">Thanos</option>
               </Input>
             </div>
-            <div className="float-right">
+            <div className="float-right" onMouseEnter={this.handleMouseEnter} onMouseLeave={this.handleMouseLeave}>
```

Comment:
Got a screenshot of what on/off hover here looks like? 🙂

Reply:
Yes, included in PR desc :)